### ISSUE TITLE
np.where prevents CUDA Tensors from compressing

### ISSUE THREAD

**Issue by dakami** [rballester/tntorch#24]
https://github.com/rballester/tntorch/blob/master/tntorch/round.py#L139

        where = np.where((torch.cumsum(S[reverse], dim=0).detach().cpu() <= delta**2))[0]

If S is located on the device, np.where causes a cpu/gpu conflict.

Replacing np.where with torch.where and removing the cpu() call allows both CPU and GPU compressed tensors to work.  Unclear if detach is necessary or if it causes resource leaks.

Cool code!
